.plzconfig

```diff
@@ -136,5 +136,11 @@ Help = Compile for the Go race detector
 Optional = true
 Help = A built target for a go.mod, which can help avoid the need to pass modules via requirements to go_repo.
 
+[PluginConfig "pkg_info"]
+Type = bool
+DefaultValue = true
+Optional = true
```

Comment:
Is this optional? If it has a default value, you don't need to specify it. Setting it to optional means you can do `PkgInfo =` without any value. I don't think that's what you want.